swagger_spec_validator/validator20.py

```diff
@@ -206,6 +210,22 @@ def validate_apis(apis, deref):
             if oper_name == 'parameters' or oper_name.startswith('x-'):
                 continue
             oper_body = deref(api_body[oper_name])
+            oper_tags = deref(oper_body.get('tags', [None]))
+
+            # Check that, if this operation has an operationId defined,
+            # no other operation with a same tag also has that
+            # operationId.
+            # import ipdb; ipdb.set_trace()
+            operation_id = oper_body.get('operationId')
+            if operation_id is not None:
+                for oper_tag in oper_tags:
+                    if operation_id in operation_tag_to_operation_id_set[oper_tag]:
+                        raise SwaggerValidationError(
+                            "Duplicate operationId: {}".format(operation_id)
+                        )
+                    else:
+                        operation_tag_to_operation_id_set[oper_tag].add(operation_id)
```

Comment:
style nit: remove the else and un-indent the add line

Reply:
donezo